node/src/main.rs

```diff
@@ -172,6 +172,17 @@ fn async_main() -> impl Future<Item = (), Error = ()> + Send + 'static {
     let ethereum_watcher = graph_datasource_ethereum::EthereumAdapter::new(
         graph_datasource_ethereum::EthereumAdapterConfig { transport },
     );
+
+    match ethereum_watcher.block_number().wait() {
+        Ok(number) => {
+            info!(logger, "Connected to Ethereum endpoint"; "block_number" => &number.as_u64())
```

Comment:
It's not clear to the user (and also not to me)  what `block_number` is. Are you sure that it always fits in an `u64`? Probably `to_string()` would be better than `as_u64()`.

Reply:
Simply logging "Connected to Ethereum node" does ensure clarity.  Given the perceived issues we have been having with the infura node not providing the latest/highest blocks I thought I would provide some context of the current state of the provided Ethereum node.

Reply:
Currently the highest block number (as displayed on etherscan.io) is 6236205, I can't imagine it will be hitting 18446744073709551615 anytime soon.

Reply:
We should either make the block_number logging more clear or remove it altogether:
`Connected to Ethereum node, current_block_number: {}`,
`Connected to Ethereum node, highest_block_number: {}`,
or simply 
`Connected to Ethereum node`

Votes?

Reply:
Ok, I get that. How about `"most_recent_block" => block_number.to_string()`, as a k/v pair?